drop-transfer/src/manager.rs

```diff
@@ -867,6 +867,7 @@ impl IncomingState {
 
                     Some(ServerReq::Fail {
                         file: file_id.clone(),
+                        msg: String::from("File failed elsewhere"),
```

Comment:
:D could we provide a better worded hint?

Reply:
we can add in additional context like "it might be due to x, y or z

Reply:
Um, the reason for this failure may literally be anything. The fun part is that this message is copied and was introduced before this PR :)
I will try to reword it a bit

Reply:
done